spec/services/organization_update_service_spec.rb

```diff
@@ -20,6 +20,53 @@
         expect(organization.reload.name).not_to eq("A brand NEW NEW name")
       end
     end
+
+    context "when organization has partners" do
+      let(:organization) do
+        create(:organization, enable_individual_requests: true, enable_child_based_requests: true, enable_quantity_based_requests: true)
+      end
+      let(:partners) { create_list(:partner, 2, organization: organization) }
+
+      before do
+        partners.first.profile.update!(
+          enable_individual_requests: true,
+          enable_child_based_requests: true,
+          enable_quantity_based_requests: false
+        )
+        partners.last.profile.update!(
+          enable_individual_requests: true,
+          enable_child_based_requests: true,
+          enable_quantity_based_requests: true
+        )
+      end
+
+      context "when all of a single partner's request flags will be disabled" do
+        before { described_class.update(organization, {enable_individual_requests: false, enable_child_based_requests: false}) }
+
+        it "should NOT change request flags in organization or its partners" do
+          aggregate_failures "request type in organization and partners" do
+            expect(organization.reload.enable_individual_requests).to eq(true)
+            expect(partners.first.profile.reload.enable_individual_requests).to eq(true)
+            expect(partners.last.profile.reload.enable_individual_requests).to eq(true)
+            expect(organization.reload.enable_child_based_requests).to eq(true)
+            expect(partners.first.profile.reload.enable_child_based_requests).to eq(true)
+            expect(partners.last.profile.reload.enable_child_based_requests).to eq(true)
```

Comment:
FYI there's a lot of extra reloading happening here that isn't necessary, since the values aren't changing in between checks. Suggestion would be
```ruby
partner1 = partners.first.reload
partner2 = partners.last.reload
```
and just use `partner1` and `partner2` in the rest of the example.

Reply:
Whoops, good spot! Done 🙂 ✅